bdsx/bds/implements.ts

```diff
@@ -1474,6 +1474,7 @@ Player.prototype.isUsingItem = procHacker.js("?isUsingItem@Player@@QEBA_NXZ", bo
 Player.prototype.hasDimension = procHacker.js("?hasDimension@Actor@@QEBA_NXZ", bool_t, { this: Player });
 Player.prototype.getAbilities = procHacker.js("?getAbilities@Player@@QEAAAEAVLayeredAbilities@@XZ", LayeredAbilities, { this: Player });
 Player.prototype.getSelectedItem = procHacker.js("?getSelectedItem@Player@@QEBAAEBVItemStack@@XZ", ItemStack, { this: Player });
+Player.prototype.getName = procHacker.js("?getName@Player@@QEBAAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@XZ", CxxString, { this:Player });
```

Comment:
please format your code with Prettier extension.

Reply:
I'm already using it..